lib/spack/docs/configuration.rst

```diff
@@ -45,13 +45,20 @@ Configuration Scopes
 -------------------------
 
 Spack pulls configuration data from files in several directories. There
-are three configuration scopes.  From lowest to highest:
+are four configuration scopes.  From lowest to highest:
 
 1. **defaults**: Stored in ``$(prefix)/etc/spack/defaults/``. These are
    the "factory" settings. Users should generally not modify the settings
    here, but should override them in other configuration scopes. The
    defaults here will change from version to version of Spack.
 
+2. **system**: Stored in ``/etc/spack``. These are settings for this
+   machine, or for all machines on which this file system is
+   mounted. The site scope can be used for settings idiosyncratic to a
+   particular machine, such as the locations of compilers or external
+   packages. These settings are presumably controlled by someone with
+   root access on the machine.
+
 2. **site**: Stored in ``$(prefix)/etc/spack/``.  Settings here affect
```

Comment:
This number is now wrong. If you use `#.` instead, it will dynamically assign the numbers: http://www.sphinx-doc.org/en/stable/rest.html#lists-and-quote-like-blocks